lib/units/device/resources/service.js

```diff
@@ -17,7 +17,7 @@ module.exports = syrup.serial()
       pathutil.vendor('STFService/wire.proto'))
 
     var resource = {
-      requiredVersion: '2.4.7'
+      requiredVersion: '2.4.8'
```

Comment:
i wouldn't mix update from origin and new feature. The commit [2e327b8bcbb60ebf504bc3d2b2efaa1ad369d939](https://github.com/wandera/stf/pull/60/commits/2e327b8bcbb60ebf504bc3d2b2efaa1ad369d939) shouldn't be part of this PR